test/empty-options.test.js

```diff
@@ -16,14 +16,14 @@ describe('when applied with no options', () => {
     const compilerEnv = pluginEnvironment.getEnvironmentStub();
     compilerEnv.context = '';
 
-    const plugin = new UglifyJsPlugin();
+    const plugin = new UglifyJsPlugin({ parallel: { cache: false, workers: 0 } });
```

Comment:
Please move all options related changes in this test to it's own test file, or to `parallel-option.test.js`. The purpose of this test is to literally have empty options.